app/helpers/script_helper.rb

```diff
@@ -2,8 +2,8 @@
 
 # rubocop:disable Rails/HelperInstanceVariable
 module ScriptHelper
-  def javascript_packs_tag_once(*names, **attributes)
-    @scripts = @scripts.to_h.merge(names.index_with(attributes))
+  def javascript_packs_tag_once(*names, **attributes_and_url_params)
```

Comment:
I'm gonna experiment with going about this a different way. It'd be nice if we could pre-embed the query parameters in the `manifest.json` file so that we don't need to assign them dynamically like this in the application.

Reply:
> I'm gonna experiment with going about this a different way. It'd be nice if we could pre-embed the query parameters in the `manifest.json` file so that we don't need to assign them dynamically like this in the application.

I took a look at this earlier and couldn't come up with an elegant solution which didn't involve either hardcoding overrides in `webpack.config.js`'s compiling of `manifest.json`, or some sort of build post-processing that manipulates the output `manifest.json`. I think it could be acceptable to have this implementation as a first-pass and follow-up with microoptimizations.